dev-python/pipenv/pipenv-2022.8.5.ebuild

```diff
@@ -58,7 +59,7 @@ distutils_enable_tests pytest
 src_prepare() {
 	local pkgName
 	local jobs=$(makeopts_jobs)
-	local packages=( attr cerberus cached_property click colorama idna iso8601 pexpect dateutil requests toml tomli urllib3 zipp )
+	local packages=( attr cerberus cached_property click colorama idna importlib_metadata importlib_resources iso8601 pexpect dateutil pyparsing requests toml tomli tomlkit urllib3 zipp )
```

Comment:
I think at this point we should break this array into multilines.
I don't mean each one it's own line, but just not use such a long line :)